test/testSamples/testSelectPrimWithoutDataSource/cube.usda

```diff
@@ -0,0 +1,25 @@
+#usda 1.0
+(
+    defaultPrim = "cube"
+)
+
+def Xform "cube" (
+    prepend apiSchemas = ["GeomModelAPI"]
+    kind = "component"
+)
+{
+    uniform bool model:applyDrawMode = 1
+    asset model:cardTextureXNeg = @./red20x20.png@
```

Comment:
All card axes have the same texture in our test, which is unimportant.  A real model would have different textures.